pkg/match/cluster.go

```diff
@@ -15,11 +15,17 @@ func toSelector(labels *metav1.LabelSelector) (labels.Selector, error) {
 	return metav1.LabelSelectorAsSelector(labels)
 }
 
-func NewClusterMatcher(clusterGroup string, clusterGroupSelector *metav1.LabelSelector, clusterSelector *metav1.LabelSelector) (*ClusterMatcher, error) {
+func NewClusterMatcher(clusterName, clusterGroup string, clusterGroupSelector *metav1.LabelSelector, clusterSelector *metav1.LabelSelector) (*ClusterMatcher, error) {
 	t := &ClusterMatcher{}
 
+	if clusterName != "" {
+		t.criteria = append(t.criteria, func(clusterNameTest, _ string, _, _ map[string]string) bool {
+			return clusterName == clusterNameTest
+		})
+	}
+
 	if clusterGroup != "" {
-		t.criteria = append(t.criteria, func(clusterGroupTest string, _, _ map[string]string) bool {
+		t.criteria = append(t.criteria, func(_, clusterGroupTest string, _, _ map[string]string) bool {
 			return clusterGroup == clusterGroupTest
```

Comment:
Should we change these "test" strings to constants?

Reply:
What does this mean? these are function parameter names.

Reply:
_facepalm_ ugh. I knew I was missing something. Disregard!